src/lib/report_graphite.js

```diff
@@ -1,16 +1,29 @@
 var moment = require('moment');
+var _ = require('lodash');
 
-// Currently doesn't work. Structure will be different, so it needs to be
-//  rewritten anyways
+module.exports = function(crawlMetrics, graphiteClient) {
 
-module.exports = function(metrics, graphiteClient) {
-  // note: crawl not defined
-  graphiteClient.write(metrics, moment(crawl.start_at).valueOf(), function(error) {
+  var metrics = {
+    crawler: {
+      publicKeyCount: Object.keys(crawlMetrics.rippleds).length,
+      connectionsCount: Object.keys(crawlMetrics.connections).length,
+      rippleds: {}
+    }
+  };
+  _.each(Object.keys(crawlMetrics.rippleds), function (rippled) {
+    metrics.crawler.rippleds[rippled] = {
+      connectionsCount: crawlMetrics.rippleds[rippled].in + crawlMetrics.rippleds[rippled].out,
+      uptime: crawlMetrics.rippleds[rippled].uptime,
+      up: 1
```

Comment:
online
`up` is `1` when the rippled is a part of the crawl and nothing when it's not